File: 2ndSemester/AlgorithimsAndDataStructuresII/Practice/Ex intro/09 - Cometa em Java/Cometa.java
import java.util.Scanner;
class Cometa{
	public static Scanner sc = new Scanner(System.in);
	public static int getProximoAno(int anoAtual){
		int proximo = 1986;//ano de passagem inicialmente forncecido
		do{	
			proximo += 76;
		}while(proximo <= anoAtual);//se esse for um ano de passagem ele eh ignorado
		
		return proximo;
	}
	public static void main(String[] args){
		int anoAtual;
		do{
			anoAtual = sc.nextInt();
			if(anoAtual != 0)
				System.out.println("Ano " + getProximoAno(anoAtual));
		}while(anoAtual != 0);
	}
}

File: 2ndSemester/AlgorithimsAndDataStructuresII/U01-WarmingUp/10 - Cometa em C-C++/cometa.c
#include <stdio.h>
#include <stdlib.h>

int getProximoAno(int anoAtual){
  int proximo = 1986;
  do{
    proximo+=76;
  }while (proximo <= anoAtual);
  return proximo;
}
int main(void){
  int anoAtual;
  do{
    scanf(" %i",&anoAtual);
      if(anoAtual != 0)
        printf("%i\n",getProximoAno(anoAtual));
  }while(anoAtual != 0);
  return 0;
}
